src/skin/legacy/tooltips.cpp

```diff
@@ -427,13 +427,41 @@ void Tooltips::addStandardTooltips() {
             << QString("%1: %2").arg(leftClick, tempoTapButton)
             << QString("%1: %2").arg(rightClick, bpmTapButton);
 
+    add("beats_set_change_marker")
+            << tr("Set Beat Marker")
+            << tr("Set a beat marker at the current play position.");
+
+    add("beats_remove_marker")
+            << tr("Remove Beat Marker")
+            << tr("Remove the beat marker at the current play position.");
```

Comment:
```suggestion
            << tr("Remove the beat change marker at the current play position.");
```

What happens with tempo and bar signature?
Maybe: 

"The tempo and bar signature before the marker is applied to the beats after the marker"

Reply:
No UI there yet, to be reviewed in #13670